components/swatch/stories/swatch.test.js

```diff
@@ -19,11 +19,46 @@ export const SwatchGroup = Variants({
 			testHeading: "Full rounding",
 			rounding: "full"
 		},
+		{
+			testHeading: "Light border",
+			borderStyle: "lightBorder",
+		},
+		{
+			testHeading: "No border",
+			borderStyle: "noBorder",
+			rounding: "none",
```

Comment:
When this test case goes through the "no color" state, how can I override the no border styles? The no color should always have a border, but this test case is specifically for "no border," so would we be looking at a CSS change for the `is-nothing` class to add a border there?